__tests__/helper.js

```diff
@@ -0,0 +1,44 @@
+const csv = require('csv-parser');
```

Comment:
could we maybe convert it one time to JSON format and do not add this parser?  
less code - less complexity - easy to support

Reply:
if you check the original file from belastingdienst, they actually provide an excel file that i converted to csv to make it easier to use... to be honest, the dutch government probably gives this information somewhere else in JSON or XML that we just need to find and download every year

Reply:
@eduardomourar could you please tell me where you got the numbers? I'm trying to extend it now for 2021

Reply:
https://www.belastingdienst.nl/wps/wcm/connect/nl/personeel-en-loon/content/hulpmiddel-loonbelastingtabellen

Reply:
@eduardomourar thank you for fast feedback, 
but it's not the data that you have in test-tax.csv's, right? 

I'm getting this table: 
<img width="1021" alt="Screenshot 2021-01-20 at 10 00 48" src="https://user-images.githubusercontent.com/1526680/105151554-622d4f00-5b06-11eb-8adb-c1e6b8bbd532.png">

Reply:
yes, it is. i just renamed the headers to reference them easier in jest

Reply:
it's not only about headers, the numbers are different. also in your test-data file we have more vectors

Reply:
the way more..
<img width="1006" alt="Screenshot 2021-01-20 at 10 14 27" src="https://user-images.githubusercontent.com/1526680/105153238-7d995980-5b08-11eb-8762-5adfb6ffbd1f.png">

Reply:
you need to download the monthly values:
![belastingdienst](https://user-images.githubusercontent.com/16357187/105164223-28b01000-5b15-11eb-9cec-8273b25a9ee3.png)

Reply:
got it, I did it per year. 
Thank you very much for your help!